services/networking/loadbalancer/taxonomy.md

```diff
@@ -0,0 +1,67 @@
+| CCC-050337  | Multiple VPC network                                  | Can be configured to handle requests to backend instances in different VPCs.                                                                                                                                                                                                                                                  |
+| CCC-050338  | External access                                       | Can configure public-facing endpoints to handle requests originating from the internet or other external networks.                                                                                                                                                                                                            |
+| CCC-050339  | Session affinity                                      | Can configure subsequent requests from an initial client to be passed to the same target.                                                                                                                                                                                                                                     |
+| CCC-050340  | Long-lived TCP connection                             | Supports connections that remain open for an extended period without teardown and reestablishment.                                                                                                                                                                                                                            |
+| CCC-050341  | Load Balancing to multiple ports on the same instance | Can load balance to multiple ports on the same instance to distribute incoming traffic across different services or applications running on the same backend server.                                                                                                                                                          |
+| CCC-050342  | Preserve Source IP address                            | Can maintain the original source IP address of incoming requests throughout the load balancing process.                                                                                                                                                                                                                       |
+| CCC-050343  | Websockets                                            | Can handle WebSocket connections.                                                                                                                                                                                                                                                                                             |
```

Comment:
Is this and 'Long-lived TCP connection' essentially the same - have I got duplication here?

Reply:
It's not the same. WebSocket support is needed when you have a chat application traffic that needs to be routed through the LB. Long-lived TCP connection is for performance optimization (like a sticky session)